src/template.js

```diff
@@ -17,8 +18,7 @@ function isInitialized() {
 }
 
 function formatValueToList(value) {
-  // `20` is the limit that the engine processes.
-  return value && value.split(',').slice(0, 20);
+  return (getType(value) === 'array' ? value : value.split(',')).slice(0, 20);
```

Comment:
We've been slicing only up to 20 here. It's another topic to tackle in another PR. In a short term, we need to split event into many and send them all if it has more than 20 objectIDs, etc.

Reply:
I would maybe move the ternary into a separate variable to increase readability: I had some difficulties parsing that line and thought the slice was only done on the `value.split()`.

Reply:
https://github.com/algolia/search-insights-gtm/pull/11/commits/b955958354a28d99a1ce04227ea95f26ea4ef912